src/main/java/io/sealights/plugins/sealightsjenkins/utils/EnvVarsInjector.java

```diff
@@ -42,7 +42,11 @@ private void printVariables() {
         logger.info("Injecting environment variables:");
         Set<String> keySet = additionalEnvVars.keySet();
         for (String key: keySet){
-            logger.info(key + " : " + additionalEnvVars.get(key));
+            if(key.equals("SL_TOKEN")){
```

Comment:
try "SL_TOKEN".equals(key) -- that way you dodge case where key==null and cause NPE